### ISSUE TITLE
Carbon Copy Recipients  in Composite Templates

### ISSUE THREAD

**Issue by bennyzr** [jondkinney/docusign_rest#129]
I only see support for carbonCopies in the method create_envelope_from_document. Is there currently support for carbonCopies recipients in the create_envelope_from_composite_template method?

**Comment by tcopeland** [comment]
@bennyzr you're right, that's not currently supported.  But seems doable - something like https://github.com/jondkinney/docusign_rest/pull/130 ?

**Comment by bennyzr** [comment]
Looks like that would work. Thanks.
